src/player.py

```diff
@@ -11,12 +11,12 @@ def __str__(self):
 
 class Player:
     def __init__(self, name):
-        self.game = None
+        self.game = None # TODO is it used?
```

Comment:
Of course, e.g. in player actions and checks.
And after all each Characrter is a part of the Game.